inline_scan.sh

```diff
@@ -284,10 +284,8 @@ prepare_inline_container() {
         CREATE_CMD+=('-e VERBOSE=true')
         RUN_CMD+=('-e VERBOSE=true')
     fi
-    if [[ "${v_flag-""}" ]]; then
-        printf '\n%s\n' "Creating volume mount -- ${VOLUME_PATH}:/anchore-engine"
-        CREATE_CMD+=('-v "${VOLUME_PATH}:/anchore-engine:rw"')
-    fi
+    printf '\n%s\n' "Creating volume mount -- ${VOLUME_PATH}:/anchore-engine"
```

Comment:
always creating the volume: below there was a `docker cp` ad hoc for the exactly opposite `if` statement.

In this way we always do the same thing